### ISSUE TITLE
Is there a way to download files inside the browser? 

### ISSUE THREAD

**Issue by SleepyPeko** [m1k1o/neko#450]
as the title says, is there a way to download files inside the virtual browser since on brave the default downloads are showing that it's pointing to /home/neko/Downloads but whenever i try to download a file inside the virtual browser it just won't let me due to "Access to local files on your machine is disabled by administraitor" and this is on brave
![image](https://github.com/user-attachments/assets/a5dec8c7-e2a8-41f7-ac50-6e405ad2eae8)
this is what happens when i try to download an image inside the virtual browser
![image](https://github.com/user-attachments/assets/8006a70c-f999-4391-9c0f-b8a208ca66ea)

this also happens with chromium and other browsers without the desktop. it doesn't allow me to download the images/videos/zip or other files to "/home/neko/Downloads". Is there a way to allow this since i don't see where to configure neko to allow the virtual browser to download the file to save in "/home/neko/Downloads" so i can remotely download from the server on the web client. 

I have noticed that chrome, chromium and brave cannot download files since there is a policy enforced? when i tested it with firefox, it just worked fine on first try. im not sure if there is a fix for chromium browsers to download files. i guess it's something with the policy set within chromium browsers that prevent them from downloading files.

Is there a way to add themes and plugins on brave and firefox? like for example i wanted to add Dark Reader (eimadpbcbfnmbkopoojfekhnkhdbieeh) but it won't allow me to do so. is there a way for me to add this in manually? like do i have to make my own docker fork just to add a custom plugin/theme in that runs alongside with brave or firefox.

My third question is that is there a way to add another user? since i already have the default admin cred set.(in the docker-compose.yml file) is there a way to do that? like i want to add a cred so my friend can access my neko server to have a watch party.

tldr: 1.) chromium, chrome and brave can't download files since admin policy? firefox works fine.
2. Is there a way to add themes and plugins on brave and firefox? in docker?
3.) Is there a way to add additional users to the config file? since i want to add my friend so we can start a watch party.

lastly this is just a suggestion but is there going to be a AMOLED theme for the GUI? 
forgot to say this but on chromium one of the shortcut links "9anime(.)to" doesn't work anymore. (i searched through and found that https://github.com/m1k1o/neko/pull/175#issuecomment-1145210597 has 9anime(.)to listed) there is a index that list all the alternatives https://theindex.moe/ and https://9animetv.to looks similar to the first domain (not the one that ends in .mor)

**Comment by yesBad** [comment]
1) https://neko.m1k1o.net/#/getting-started/?id=allow-file-uploading-amp-downloading
2) https://neko.m1k1o.net/#/getting-started/?id=want-to-customize-and-install-own-add-ons-set-custom-bookmarks
3) You don't add more users, everyone can log-in with a random username + user password.

**Comment by SleepyPeko** [comment]
> 1. https://neko.m1k1o.net/#/getting-started/?id=allow-file-uploading-amp-downloading
> 2. https://neko.m1k1o.net/#/getting-started/?id=want-to-customize-and-install-own-add-ons-set-custom-bookmarks
> 3. You don't add more users, everyone can log-in with a random username + user password.
> 
> Most answers are found in the documentation. Read it 😛

Damn, I might just overlooked it when reading the docs. Had this issue opened too much. Thanks tho
